src/main/resources/com/mathworks/ci/MatlabInstallationAxis/help.html

```diff
@@ -0,0 +1,11 @@
+<div>
+    <p>
+        Use this list to add MATLAB versions as axes to Matrix configuration.
```

Comment:
Moving text around along with some other edits. Here is what I suggest:

Use this list to add MATLAB versions as axis values to your matrix configuration. The list includes the names of all MATLAB versions that have been registered as a tool in Jenkins. <br>

In each build iteration, the plugin prepends one of the selected versions to the PATH environment variable and invokes it to perform the build. <br>

Reply:
Done.